### PR TITLE
Set up easy linting

### PR DESCRIPTION
**Pull request by ConnorChristensen**
While most people probably have ESlint integration into VScode or other editor, this is an easy way to check the whole project at once, and makes it more accessible for those folks that prefer the terminal to keep their work in-line with the linter.

### CODE REVIEW

**Review by ivanasetiawan — APPROVED** [review]

File: src/stories/ui-label.stories.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ export default {
     component: UiLabel,
 };
 
-const Template = args => ({
```
Comment (bartderijk) [inline src/stories/ui-label.stories.js:8]: @ConnorChristensen  @ivanasetiawan I'm pretty sure this change causes our code examples in storybook to loose their formatting. While I really appreciate quick follow-ups on PR's, please hold off on merging until others had a chance to look at the changes as well. 10 minutes from opening the PR until merging is a little too eager :)

Comment (ivanasetiawan) [reply src/stories/ui-label.stories.js:8]: @bartderijk You're right! I was way too eager.. My apology. Shall revert and re-create this branch?

Comment (bartderijk) [reply src/stories/ui-label.stories.js:8]: No worries, I found that almost all code examples are broken. I'll create a PR to fix them all in one go.

Comment (ivanasetiawan) [reply src/stories/ui-label.stories.js:8]: Thank you @bartderijk - I will keep this in mind next time.

**Comment by jpsc** [comment]
Not really sure if this was required. 

Sure, it's just an added script and it's minimal in impact but we are doing the linting and fixing on commit automatically, so I'm not sure why would you need it.

File: src/stories/ui-label.stories.js (hunk 1 above)
Comment (jpsc) [reply src/stories/ui-label.stories.js:8]: > 10 minutes from opening the PR until merging is a little too eager :)

Yep 😇

Comment (ConnorChristensen) [reply src/stories/ui-label.stories.js:8]: I think that my linting change does not break the formatting, but I would be happy to help investigate the cause. I also agree that there is no urgency to get a change like this merged in.

**Comment by ConnorChristensen** [comment]
I was reviewing #636 and some of the committed code did not follow the linting rules, so I figured it could be a help to the team to be able to run the linter manually without having to use `npx`. It looks like the husky configuration on commit does have linting for commit messages, but no rules for linting the source code:

```sh
#!/bin/sh
. "$(dirname "$0")/_/husky.sh"

npx --no-install commitlint --edit "$1"
```

Maybe this was something the repo had but it has been turned off or broken?

**Comment by jpsc** [comment]
Most likely. I've cloned the repo and encountered the same behavior. 
I would prefer to fix this than to have that script.

File: src/stories/ui-label.stories.js (hunk 1 above)
Comment (bartderijk) [reply src/stories/ui-label.stories.js:8]: Like discussed, indeed it was already broken. Thanks for checking, I'll look into a fix.

**Pull request merged**